services/main/static-asset-server.mjs

```diff
@@ -31,10 +31,13 @@ class StaticAssetServer {
    */
   initialize() {
     // TODO: Restrict the visible pages.
-    this.app.use('/static', express.static(__dirname+'/../../'));
+    // Not sure if all static files are in sites
+    this.app.use('/static', express.static(__dirname+'/../../sites'));
```

Comment:
We'll need common/maplib/map.mjs, which will be unavailable if we only export sites directory.